src/components/AutocompleteLocation.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <div class="columns">
+    <GmapPlaceInput
+    placeholder='Enter location You are searching for...'
+    v-on:place_changed='placeChange'
+    >
+    </GmapPlaceInput>
+    </div>
+</template>
+
+<script>
+import Store from '../store/store';
+
+export default
+{
+    name: 'autocomplete-location',
+    methods:
+    {
+        placeChange (e)
```

Comment:
As a part of convention as I have  e -> event, i -> iterator, n -> numeric value or nth value, d -> data element of data arr, x -> x coords, y -> y coords, z -> z coords or elements of xyz matrix,
so event this will save me time really to do this this way.